flickr_download/flick_download.py

```diff
@@ -113,14 +113,21 @@ def download_set(set_id, get_filename, size_label=None):
 
     for photo in photos:
         fname = get_full_path(dirname, get_filename(pset, photo, suffix))
+        if 'video' in photo.getInfo():
```

Comment:
Do all the video specific logic in one go here, by both fixing the file name and setting the `size_label` to the video one.